count-two-dets

```diff
@@ -1,28 +1,26 @@
 Count([det1, det2], 2)
-stage: (reader: det1), (), {}
-stage: (reader: det2), (), {}
-open_run: (None), (), {'plan_type': 'Count', 'plan_args': {'delay': '0', 'detectors': '[reader: det1, reader: det2]', 'num': '2'}, 'detectors': ['det1', 'det2']}
-wait: (None), ('_flyers',), {}
+open_run: (None), (), {'detectors': ['det1', 'det2'], 'plan_args': {'num': 2, 'detectors': '[reader: det1, reader: det2]'}}
 checkpoint: (None), (), {}
 create: (None), (), {'name': 'primary'}
-trigger: (reader: det1), (), {'block_group': 'A'}
-trigger: (reader: det2), (), {'block_group': 'A'}
-wait: (None), ('A',), {}
+stage: (reader: det1), (), {}
```

Comment:
I have no evidence for it or specific cases in mind, but I have a feeling that the lazy-staging will cause problems (or at the very least confusion) later down the line.